CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # TBD
 ### Changed
 * Clearer replica-check exception message. See [#47](https://github.com/HotelsDotCom/circus-train/issues/47).
+### Fixed
+* S3-S3 Hive Diff calculating incorrect checksum on folders
```

Comment:
Add a See like above with a link to the ticket it fixes (and a period at the end ;))